Index: setup.py
===================================================================
--- setup.py	(revision 1001)
+++ setup.py	(working copy)
@@ -1,19 +1,21 @@
-from ez_setup import use_setuptools
-use_setuptools()
-from setuptools import setup, find_packages
+#!/usr/bin/env python
 
+from distutils.core import setup
+
 setup(name = "SQLAlchemy",
     version = "0.3.7",
     description = "Database Abstraction Library",
     author = "Mike Bayer",
     author_email = "mike_mp@zzzcomputing.com",
     url = "http://www.sqlalchemy.org",
-    packages = find_packages('lib'),
-    package_dir = {'':'lib'},
-    entry_points = { 
-      'sqlalchemy.databases': [
-        '%s = sqlalchemy.databases.%s:dialect' % (f,f) for f in 
-          ['sqlite', 'postgres', 'mysql', 'oracle', 'mssql', 'firebird']]},
+    packages = ['sqlalchemy', 'sqlalchemy.databases', 'sqlalchemy.orm', 
+                'sqlalchemy.ext', 'sqlalchemy.engine', 'sqlalchemy.mods'],
+    package_dir={'sqlalchemy': 'lib/sqlalchemy',
+                 'sqlalchemy.databases': 'lib/sqlalchemy/databases',
+                 'sqlalchemy.orm': 'lib/sqlalchemy/orm',
+                 'sqlalchemy.ext': 'lib/sqlalchemy/ext',
+                 'sqlalchemy.engine': 'lib/sqlalchemy/engine',
+                 'sqlalchemy.mods': 'lib/sqlalchemy/mods'},
     license = "MIT License",
     long_description = """\
 SQLAlchemy is:
@@ -57,3 +59,4 @@
 
 
 
+
