--- bin/pkgorder.orig	2008-06-25 15:50:50.000000000 +0800
+++ bin/pkgorder	2008-06-25 15:50:58.000000000 +0800
@@ -8,10 +8,10 @@
 
 # FIXME: we really want this information somewhere, but not in the pkgorder
 # file itself.
-from rhpl.log import log
-from anaconda_log import anaconda_log
-anaconda_log.open("/dev/null")
-log.handler = anaconda_log
+#from rhpl.log import log
+#from anaconda_log import anaconda_log
+#anaconda_log.open("/dev/null")
+#log.handler = anaconda_log
 
 # set up rpm dependency whiteouts
 import whiteout
