Index: Makefile.in
===================================================================
--- Makefile.in	(revision 474)
+++ Makefile.in	(working copy)
@@ -30,7 +30,7 @@
 prefix = @prefix@
 exec_prefix = @exec_prefix@
 # Pathname of directory to install the binary.
-BINDIR = @sbindir@
+BINDIR = @bindir@
 # Pathname of directory to install the man page.
 MANDIR = @mandir@
 # Pathname of directory to install the CGI programs.
@@ -74,7 +74,7 @@
 
 SUBDIRS =	cgi-src extras
 
-all:		this subdirs
+all:		this
 this:		$(ALL)
 
 thttpd: $(OBJ)
@@ -106,7 +106,7 @@
 	) ; done
 
 
-install:	installthis install-man installsubdirs
+install:	installthis 
 
 installthis:
 	-mkdir -p $(DESTDIR)$(BINDIR)
