--- SetupConfig.py.orig	2008-06-16 16:53:08.000000000 +0800
+++ SetupConfig.py	2008-06-16 16:53:23.000000000 +0800
@@ -36,14 +36,7 @@
 import os.path
 from distutils.core import Extension
 
-## we only assume the presence of a c compiler on Posix systems, NT people will
-#  have to enable this manually. 
-if os.name == 'posix':
-    ext_modules=[Extension("Cheetah._namemapper", [os.path.join("src" ,"_namemapper.c")]
-                           )
-                 ]
-else:
-    ext_modules=[]
+ext_modules=[]
 
 
 ## Data Files and Scripts
