--- pdsh-2.14/src/pdsh/rcmd.c	2007-06-07 00:18:53.000000000 +0800
+++ pdsh-2.14/src/pdsh/rcmd.c.make_ssh_default	2007-08-16 15:23:21.000000000 +0800
@@ -45,7 +45,7 @@
  *   Warning: If none of these modules are loaded, there will be no default.
  */
 static char * rcmd_rank[] = 
-    { "mrsh", "rsh", "ssh", "krb4", "qsh", "mqsh", "exec", "xcpu", NULL };
+    { "ssh", "mrsh", "rsh", "krb4", "qsh", "mqsh", "exec", "xcpu", NULL };
 
 struct rcmd_module {
     char *              name;
