--- path.py.orig	2008-07-03 22:37:32.000000000 +0800
+++ path.py	2008-07-03 22:39:52.000000000 +0800
@@ -73,6 +73,34 @@
 if hasattr(file, 'newlines'):
     _textmode = 'U'
 
+    
+import sys
+version_info = sys.version_info[:2]
+ 
+#only affects python 2.3.X. Taken from python 2.4.3
+if os.path.__name__ == 'posixpath' and version_info < (2,4):
+    from posixpath import *
+    def _resolve_link(path):
+        """Internal helper function.  Takes a path and follows symlinks
+        until we either arrive at something that isn't a symlink, or
+        encounter a path we've seen before (meaning that there's a loop).
+        """
+        paths_seen = []
+        while islink(path):
+            if path in paths_seen:
+                # Already seen this path, so we must have a symlink loop
+                return None
+            paths_seen.append(path)
+            # Resolve where the link points to
+            resolved = os.readlink(path)
+            if not isabs(resolved):
+                dir = dirname(path)
+                path = normpath(join(dir, resolved))
+            else:
+                path = normpath(resolved)
+        return path
+
+    os.path._resolve_link = _resolve_link
 
 class TreeWalkWarning(Warning):
     pass
